reviews/sandbox/2022-wasmedge-annual.md

```diff
@@ -0,0 +1,158 @@
+* The OpenGauss and libsql communities used WasmEdge to power UDFs (User Defined Functions) in database queries.
+* Apache Traffic Server uses WasmEdge as a plugin to process the traffic flow.
+
+Another highlighted collaboration is that Docker Desktop integrates WasmEdge to support Wasm containers.
+
+Furthermore, the WasmEdge project is used by numerous commercial companies including Docker, VMWare, FutureWei, Huawei, 5miles, and ByteDance. You can check out the full list [here](https://wasmedge.org/docs/contribute/users).
+
+## Project goals
+
+This is the WasmEdge project's second Annual review after joining the CNCF Sandbox.
+
+For the immediate future, WasmEdge will focus on the following areas.
+
+* Technical goals
+    * Support more WebAssembly proposals and specs, such as threading, stack switching, logging, and debugging. In particular, we aim to be one of the first Wasm runtimes to fully support the WebAssembly component model spec.
+    * Improve support for microservices. We will enable WasmEdge applications to access commonly used databases, messaging services, proxies, sidecars, and other services.
+    * Improve support for data streaming functions. We will work with data streaming partners to support embedded functions in WasmEdge in those frameworks.
+    * Improve support for AI inference workloads in Wasm. We will continue adding support for popular models in computer vision, voice recognition, and document AI. We will also enable inference on LLMs such as Llama.
+    * Support AI training on Wasm. We are collaborating with ByteDance and the Ray community to support WasmEdge on Ray nodes.
+    * Support the complete Node.js API and NPM module ecosystem in WasmEdge's JavaScript runtime.
+    * Improve WasmEdge support in popular application frameworks such as Dapr, Spin, etc.
+    * Better support for host SDKs in Rust, Go, Java, and Python for embedding WasmEdge in other applications.
+    * Debugging, monitoring, and observability tools for WasmEdge.
+    * Further optimize the developer experience and user experience for WasmEdge plugins. That would make it easy for the community to add new features to WasmEdge.
+
+
+Our 2023 roadmap could be found [here](https://github.com/WasmEdge/WasmEdge/blob/master/docs/ROADMAP.md).
+
+* User adoption goals
+    * Drive adoption through integrating more open-source projects. 
+    * Continue working closely with end-users to bring WasmEdge into production systems in at least 3-5 commercial companies.
+
+* Community and community collaborations
+    * Recruit and support proposal maintainers from more companies other than Second State.
+    * More community collaborations with open source projects in Linux Foundation and Apache Foundation.
+    * Attending continuously the mentorship programs like Google Summmer of Code, Google Season of Docs, LF mentorship and Open Source of Summer to drive community contributions by students.
```

Comment:
Hi @alabulei1 what is Open Source of Summer?

Reply:
Hi  @shivaylamba  I added some links for each program. Open Source of Summer should be Open Source Program Pan. I made a typo here.